src/main/resources/db/changelog.xml

```diff
@@ -11,6 +11,12 @@
     <property name="identifier_type" value="varchar(63)"/>
     <property name="description_type" value="varchar(2047)"/>
 
+    <changeSet author="jh" id="db-init">
+        <sql dbms="postgresql" >
+            CREATE EXTENSION IF NOT EXISTS pgcrypto;
```

Comment:
I tried that and it did not work for me, because the migration is running as `drmanager` and that account does not have (and should not have) superuser. You have to be superuser to add extensions.

Reply:
Hmm, good point. I'm going to remove this and consider this the responsibility of the user that sets up the database (I am putting together a script based on my experience getting dev/integration set up). I am concerned because this works fine for me locally and on dev, meaning the `drmanager` user has the wrong privileges. I'll look into this separately.

Reply:
When I run the db/create... script, I always am doing it as postgres. That script is creating the extension.